pkg/bootstrap/api/types.go

```diff
@@ -80,6 +88,11 @@ const (
 	// authenticate as. The full username given is "system:bootstrap:<token-id>".
 	BootstrapUserPrefix = "system:bootstrap:"
 
-	// BootstrapGroup is the group bootstrapping bearer tokens authenticate in.
-	BootstrapGroup = "system:bootstrappers"
```

Comment:
Have you looked up and replaced all occurances of this const? I _think_ there should be some more places...